modules/hcp-aws-deploy-hc.adoc

```diff
@@ -18,21 +18,22 @@ You can deploy a hosted cluster on {aws-first} by using the `hcp` command-line i
 
 .Procedure
 
-* To create the hosted cluster on {aws-short}, run the following command:
+* To create a hosted cluster on {aws-short}, run the following command:
 +
 [source,terminal]
 ----
 $ hcp create cluster aws \
-    --name <hosted_cluster_name> \ <1>
-    --infra-id <infra_id> \ <2>
-    --base-domain <basedomain> \ <3>
-    --sts-creds <path_to_sts_credential_file> \ <4>
-    --pull-secret <path_to_pull_secret> \ <5>
-    --region <region> \ <6>
+    --name <hosted_cluster_name> \// <1>
+    --infra-id <infra_id> \// <2>
+    --base-domain <basedomain> \// <3>
+    --sts-creds <path_to_sts_credential_file> \// <4>
+    --pull-secret <path_to_pull_secret> \// <5>
+    --region <region> \// <6>
     --generate-ssh \
-    --node-pool-replicas <node_pool_replica_count> \ <7>
-    --namespace <hosted_cluster_namespace> \ <8>
-    --role-arn <role_name> <9>
+    --node-pool-replicas <node_pool_replica_count> \// <7>
+    --namespace <hosted_cluster_namespace> \// <8>
+    --role-arn <role_name> \// <9>
+    --render <10>
```

Comment:
```suggestion
    --render <10> > somefile.yaml
```
you would need to redirect the output to a file when using the `render` flag, or use `--render-into somefile.yaml`